scripts/test/setup-integration-infra.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const awsRequest = require('@serverless/test/aws-request');
+const fs = require('fs');
+const path = require('path');
+const { SHARED_INFRA_TESTS_CLOUDFORMATION_STACK } = require('../../test/utils/cludformation');
+
+(async () => {
+  process.stdout.write('Starting setup of integration infrastructure...\n');
+  const cfnTemplate = fs.readFileSync(path.join(__dirname, 'cloudformation.yml'), 'utf8');
+  const kafkaServerProperties = fs.readFileSync(path.join(__dirname, 'kafka.server.properties'));
+
+  process.stdout.write('Checking if integration tests CloudFormation stack already exists...\n');
+  try {
+    await awsRequest('CloudFormation', 'describeStacks', {
+      StackName: SHARED_INFRA_TESTS_CLOUDFORMATION_STACK,
+    });
+    process.stdout.write('Integration tests CloudFormation stack already exists. Quitting.\n');
+    return;
+  } catch (e) {
+    process.stdout.write('Integration tests CloudFormation does not exist. Continuing.\n');
```

Comment:
I would inspect an _error_. As we do not have the guarantee it's exactly this case (in all other I think we should just let error surface, so rethrow)